pubspec.yaml

```diff
@@ -1,6 +1,6 @@
 name: flutter_blue
 description: Bluetooth plugin for Flutter
-version: 0.5.0
+version: 0.6.0
```

Comment:
```suggestion
version: 0.5.0
```

Let @pauldemarco decide, to which version he want's to update it.

Reply:
https://semver.org/

Reply:
> https://semver.org/

I just ment that if everyone would change the version in his PR, then there would be a hell of merge conflicts. This is why it is easier to let the author decide wether he's updating the version of the library or not. not every PR needs a version change.

Reply:
Ah, ok